src/reaction_lammps_mupt/reaction_template_builder/reaction_template_pipeline/util.py

```
@@ -0,0 +1,178 @@
+        dict: A dictionary containing aggregated strings for:
+              - "reactions_names"
+              - "smart_references"
+              - "mechanism_references"
+    """
+    reactions_names = ""
+    smart_references = ""
+    mechanism_references = ""
+    formatted_dict = {}
+    data_smiles_list = []
+
+    for key, reaction in detected_reactions.items():
```

Ruff (0.14.14)

[warning] Loop control variable `key` not used within loop body

Rename unused `key` to `_key`

(B007)